spec/integration/normalizer_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe "Parser" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.